src/main/resources/assets/presencefootsteps/lang/zh_tw.json

```diff
@@ -1,63 +1,86 @@
 {
-	"key.presencefootsteps.settings": "臨場腳步聲",
-	"pf.default_sounds.name": "預設聲音包",
-	"menu.pf.title": "臨場腳步聲選項",
-	"menu.pf.multiplayer.true": "單人 + 多人遊戲",
-	"menu.pf.multiplayer.false": "單人遊戲",
-	"menu.pf.global.all": "實體：全部",
-	"menu.pf.global.players_and_hostiles": "實體：玩家 + 敵對",
-	"menu.pf.global.players_only": "實體：僅玩家",
-	"menu.pf.volume.min": "全域音量：關閉",
-	"menu.pf.volume": "全域音量：%d%%",
-	"menu.pf.volume.tooltip": "臨場腳步聲所有聲音的全域音量。如果你想完全停用此模組，請設為 0。",
-	"menu.pf.volume.player": "用戶端玩家：%d%%",
-	"menu.pf.volume.player.tooltip": "改變自己腳步聲的音量。",
-	"menu.pf.volume.other_players": "其他玩家：%d%%",
-	"menu.pf.volume.other_players.tooltip": "在多人遊戲中改變其他玩家腳步聲的音量。",
-	"menu.pf.volume.running": "跑步增加音量：%d%%",
-	"menu.pf.volume.running.tooltip": "跑步時腳步聲音量增加多少。\n\n這個值將「加入」到總音量中，會隨著你的速度而縮放。\n\n建議先將其設為較低的值，然後再根據需求進行調整。較高的值會使效果更明顯。",
-	"menu.pf.volume.wet": "潮濕的表面：%s",
-	"menu.pf.volume.wet.tooltip": "變更在潮濕方塊上行走時產生的音量。",
-	"menu.pf.stance": "站姿：%s",
-	"menu.pf.stance.auto": "自動",
-	"menu.pf.stance.none": "無",
-	"menu.pf.stance.none.tooltip": "會根據 Mine Little Pony 模組自動選擇合適的站姿。如果沒有安裝此模組，預設與雙足動物行為相同",
-	"menu.pf.stance.quadruped": "四足動物",
-	"menu.pf.stance.quadruped.tooltip": "四足動物：模擬四條腿的走路方式",
-	"menu.pf.stance.biped": "雙足動物",
-	"menu.pf.stance.biped.tooltip": "雙足動物：模擬兩條腿（人類）的走路方式",
-	"menu.pf.stance.flying": "飛馬",
-	"menu.pf.stance.flying.tooltip": "飛馬：模擬四條腿帶翅膀的走路方式",
-	"menu.pf.stance.flying_biped": "禽鳥",
-	"menu.pf.stance.flying_biped.tooltip": "禽鳥：模擬兩條腿帶翅膀的走路方式",
-	"menu.pf.report.full": "完整報告",
-	"menu.pf.report.concise": "簡潔報告",
-	"pf.report.save": "檔案儲存為：%s",
-	"pf.report.error": "生成報告失敗：%s",
-	"pf.update.title": "有可用的更新",
-	"pf.update.text": "臨場腳步聲 %s",
-	"pf.update.checking": "正在檢查更新...",
-	"pf.update.updates_available": "有可用的更新！\n臨場腳步聲 %s 適用於 Minecraft %s",
-	"pf.update.up_to_date": "你使用的已是最新版",
-	"subtitles.pf.footsteps": "腳步聲",
-	"subtitles.pf.wood_squeak": "木頭嘎吱聲",
-	"subtitles.pf.wood_bends": "木頭彎曲聲",
-	"subtitles.pf.wood_creaks": "木頭碰撞聲",
-	"subtitles.pf.swim": "水流聲",
-	"subtitles.pf.wings_flap": "翅膀拍打聲",
-	"subtitles.pf.leaves_rustle": "樹葉沙沙聲",
-	"subtitles.pf.grass_rustles": "草摩擦聲",
-	"subtitles.pf.land": "翅膀合上聲",
-	"subtitles.pf.metal_clink": "金屬鏗鏘聲",
-	"subtitles.pf.snow_crunch": "雪擠壓聲",
-	"subtitles.pf.sand_crunch": "沙子摩擦聲",
-	"subtitles.pf.grass_crunch": "草踩踏聲",
-	"subtitles.pf.gravel.crunch": "礫石碰撞聲",
-	"subtitles.pf.ice_crack": "冰塊碎裂聲",
-	"subtitles.pf.mud_squealk": "泥巴踩踏聲",
-	"subtitles.pf.glass_rattles": "玻璃踩踏聲",
-	"subtitles.pf.glass_croak": "玻璃彎曲低鳴聲",
-	"subtitles.pf.chain": "鎖鏈晃動聲",
-	"subtitles.pf.stone_crack": "石頭碎裂聲",
-	"subtitles.pf.wax_squeak": "蠟吱吱聲"
-}
+    "mod.presencefootsteps.name": "臨場腳步聲 Presence Footsteps",
```

Comment:
Can you fix the indentation please? Should be 1 tab and the same for the entire file.

Reply:
Okay , I fixed the indentation
thanks you